src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1419,11 +1419,17 @@ def _configure_security_policy_storage_params(arg_ctx):
     with self.argument_context('sql midb restore') as c:
         create_args_for_complex_type(
             c, 'parameters', ManagedDatabase, [
+                'deleted_time',
```

Comment:
`deleted_time` does not align with other apis/clients as well. I think it should be `dropped_time`?

Reply:
Deleted-time is correct: https://docs.microsoft.com/en-us/cli/azure/sql/db?view=azure-cli-latest#az-sql-db-restore

Consistency with an experience (e.g.: AzCLI, PowerShell, or Portal) is higher priority than consistency across experiences.

Reply:
Note that I deliberately chose to use `--deleted-time` when implementing `az sql db restore` because the command is `az sql db delete`, not `az sql db drop`.

Reply:
Makes sense and I agree on consistency throughout any one given client. Thanks for the extra context!